Doc/Tutorial/chapter_introduction.tex

```diff
@@ -314,4 +314,4 @@ \section{Frequently Asked Questions (FAQ)}
 
 \end{enumerate}
 
-\noindent For more general questions, the Python FAQ pages \url{http://www.python.org/doc/faq/} may be useful.
+\noindent For more general questions, the Python FAQ pages \url{https://docs.python.org/2/faq/index.html} may be useful.
```

Comment:
I propose use references to Python 2 documentation (Python 2 is not obsolete, but it will be in 2020) because some examples do not work in Python 3 and reference to urllib documentation in chapter_entrez.tex should be to Python 2 documentation (urllib module in Python 3 has 4 modules). I propose to open WIP (work in progress) pull request to completely rewrite Tutorial to Python 3.